src/gtc/gtcpp/gtcpp.py

```diff
@@ -173,19 +170,18 @@ class GTParamList(LocNode):
     accessors: List[GTAccessor]
 
 
-class GTFunctor(LocNode, SymbolTableTrait):
-    name: SymbolName
+class GTFunctor(LocNode, eve.SymbolTableTrait):
+    name: eve.Coerced[eve.SymbolName]
     applies: List[GTApplyMethod]
     param_list: GTParamList
 
 
 class Arg(LocNode):
-    name: SymbolRef
+    name: eve.Coerced[eve.SymbolRef]
 
-    class Config(eve.concepts.FrozenModel.Config):
-        pass
+    # class Config(eve.concepts.FrozenModel.Config):
+    #     pass
```

Comment:
```suggestion
```

Reply:
I'll roll this in with the next set of review changes tomorrow.